eq-author/src/components/preview/Answers/__snapshots__/DateRangeAnswer.test.js.snap

```diff
@@ -9,6 +9,7 @@ exports[`DateRangeAnswer should render 1`] = `
         "properties": Object {
           "format": "dd/mm/yyyy",
         },
+        "secondaryLabel": "to",
```

Comment:
Should this secondaryLabel be sent down?

Reply:
secondaryLabel now removed from being passed down to DataAnswer